code/modules/clothing/suits/wintercoats.dm

```diff
@@ -262,6 +279,8 @@
 		/obj/item/reagent_containers/pill,
 		/obj/item/reagent_containers/syringe,
 		/obj/item/storage/pill_bottle,
+		/obj/item/storage/bag/bio,
+
```

Comment:
remove this newline

Reply:
Fixed, but will post it later as I have no Ethernet currently and writing from phone